pkg/didcomm/dispatcher/outbound.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package dispatcher
+
+import (
+	"errors"
+
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
+)
+
+// OutboundDispatcher dispatch msgs to destination
+type OutboundDispatcher struct {
+	outboundTransport []transport.OutboundTransport
```

Comment:
Consider instead just accepting a `func(scheme string) (transport.OutboundTransport, error)`. That way this OutboundDispatcher doesn't have to search through them itself - a nice separation of concerns.